### DISCUSSION TITLE
[Feature Request]:能否添加一个电视剧全集订阅功能？

### DISCUSSION THREAD

**Discussion by LeeHK1989** [jxxghp/MoviePilot#3461] [讨论]
### 当前程序版本

1.8.0

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

比较喜欢完结了下载全集，不想一集一集下，能否添加一个全集下载功能？

**Comment by cddjr** [comment]
也许可以添加关键字实现，比如“完结|全集”

**Comment by LeeHK1989** [comment]
> 也许可以添加关键字实现，比如“完结|全集”

我目前就是这样操作的。但是关键词就过滤不了其他词了

**Comment by cddjr** [comment]
> > 也许可以添加关键字实现，比如“完结|全集”
> 
> 我目前就是这样操作的。但是关键词就过滤不了其他词了

有“或”又有“与”的逻辑关系需要使用正则了，举个例子
`(?=.*(全集|完结))(?=.*HHWEB)`
代表筛选出含有全集或者完结并且还得是HHWEB的官种。

正则比较复杂，我也不熟，这个例子也只是我随便写的，不一定是最好的写法

**Comment by LeeHK1989** [comment]
> > > 也许可以添加关键字实现，比如“完结|全集”
> > 
> > 
> > 我目前就是这样操作的。但是关键词就过滤不了其他词了
> 
> 有“或”又有“与”的逻辑关系需要使用正则了，举个例子 `(?=.*(全集|完结))(?=.*HHWEB)` 代表筛选出含有全集或者完结并且还得是HHWEB的官种。
> 
> 正则比较复杂，我也不熟，这个例子也只是我随便写的，不一定是最好的写法

太牛逼了 感谢 我去试试
